.changeset/brave-scripts-dance.md

```diff
@@ -0,0 +1,47 @@
+---
+"@biomejs/biome": patch
+---
+
+Added the new nursery rule [`useRequiredScripts`](https://biomejs.dev/linter/rules/use-required-scripts/), which enforces the presence of required scripts in `package.json`. This rule is particularly useful in monorepo environments where consistency across workspaces is important.
```

Comment:
The changeset is not documentation. Please keep it brief.

If you could truncate it to just this line, and maybe clarify it by saying something along the lines of "enforces presence of entries in the `scripts`​ section in `package.json`​ files, which is  configurable"